src/med_bench/mediation.py

```diff
@@ -613,177 +443,21 @@ def mediation_multiply_robust(
-        )
-
-        # predict E[Y|T=t,M=m,X]
-        mu_t0_m0[test_index] = y_reg.predict(
-            get_interactions(interaction, x, t0, m0)[test_index, :]
-        )
-        mu_t0_m1[test_index] = y_reg.predict(
-            get_interactions(interaction, x, t0, m1)[test_index, :]
-        )
-        mu_t1_m1[test_index] = y_reg.predict(
-            get_interactions(interaction, x, t1, m1)[test_index, :]
-        )
-        mu_t1_m0[test_index] = y_reg.predict(
-            get_interactions(interaction, x, t1, m0)[test_index, :]
-        )
-
-        # E[E[Y|T=1,M=m,X]|T=t,X] model fitting
-        reg_y_t1m1_t0 = RidgeCV(alphas=alphas, cv=CV_FOLDS).fit(
-            x[test_index, :][ind_t0, :], mu_t1_m1[test_index][ind_t0]
-        )
-        reg_y_t1m0_t0 = RidgeCV(alphas=alphas, cv=CV_FOLDS).fit(
-            x[test_index, :][ind_t0, :], mu_t1_m0[test_index][ind_t0]
-        )
-        reg_y_t1m1_t1 = RidgeCV(alphas=alphas, cv=CV_FOLDS).fit(
-            x[test_index, :][~ind_t0, :], mu_t1_m1[test_index][~ind_t0]
-        )
-        reg_y_t1m0_t1 = RidgeCV(alphas=alphas, cv=CV_FOLDS).fit(
-            x[test_index, :][~ind_t0, :], mu_t1_m0[test_index][~ind_t0]
-        )
-
-        # predict E[E[Y|T=1,M=m,X]|T=t,X]
-        E_mu_t1_t0[test_index] = (
-            reg_y_t1m0_t0.predict(x[test_index, :]) * f_00x[test_index]
-            + reg_y_t1m1_t0.predict(x[test_index, :]) * f_01x[test_index]
-        )
-        E_mu_t1_t1[test_index] = (
-            reg_y_t1m0_t1.predict(x[test_index, :]) * f_10x[test_index]
-            + reg_y_t1m1_t1.predict(x[test_index, :]) * f_11x[test_index]
-        )
-
-        # E[E[Y|T=0,M=m,X]|T=t,X] model fitting
-        reg_y_t0m1_t0 = RidgeCV(alphas=alphas, cv=CV_FOLDS).fit(
-            x[test_index, :][ind_t0, :], mu_t0_m1[test_index][ind_t0]
-        )
-        reg_y_t0m0_t0 = RidgeCV(alphas=alphas, cv=CV_FOLDS).fit(
-            x[test_index, :][ind_t0, :], mu_t0_m0[test_index][ind_t0]
-        )
-        reg_y_t0m1_t1 = RidgeCV(alphas=alphas, cv=CV_FOLDS).fit(
-            x[test_index, :][~ind_t0, :], mu_t0_m1[test_index][~ind_t0]
-        )
-        reg_y_t0m0_t1 = RidgeCV(alphas=alphas, cv=CV_FOLDS).fit(
-            x[test_index, :][~ind_t0, :], mu_t0_m0[test_index][~ind_t0]
-        )
-
-        # predict E[E[Y|T=0,M=m,X]|T=t,X]
-        E_mu_t0_t0[test_index] = (
-            reg_y_t0m0_t0.predict(x[test_index, :]) * f_00x[test_index]
-            + reg_y_t0m1_t0.predict(x[test_index, :]) * f_01x[test_index]
-        )
-        E_mu_t0_t1[test_index] = (
-            reg_y_t0m0_t1.predict(x[test_index, :]) * f_10x[test_index]
-            + reg_y_t0m1_t1.predict(x[test_index, :]) * f_11x[test_index]
-        )
+    classifier_y, cross_y_clf, classifier_m, classifier_x = get_y_m_x_classifiers(regularization, forest, calibration, calib_method)
+    p_x, f, mu, cross_mu = estimate_f_mu_cross_mu(t, m, x, y, crossfit, classifier_y, cross_y_clf, classifier_m, classifier_x, interaction)
```

Comment:
You have to respect the 80 characters limit. Please use black or other linters to improve formatting

Reply:
All lines exceeding 80 characters were addressed in commit 6b96b6c2c3c89da32723fdf20b315f6fb11b80e6